sbi/utils/torchutils.py

```diff
@@ -265,5 +265,33 @@ def atleast_2d(t: Tensor) -> Tensor:
     return t if t.ndim >= 2 else t.reshape(1, -1)
 
 
+def maybe_add_batch_dim_to_size(t: torch.Size) -> torch.Size:
```

Comment:
this adds a batch dim if only if `len(t)<=1` no? 

You could write this is in the docstring and reduce the examples because then it is clear that for any other `Size` it will just be the identity function, right?

Reply:
Yes, you're right, doing this.